sql/dedup_cases_raw.sql

```diff
@@ -0,0 +1,10 @@
+USE `nlrb_data`;
+
+DROP TABLE IF EXISTS `cases_raw_deduped`;
```

Comment:
instead of the `DROP TABLE IF EXISTS...; CREATE TABLE` format, the `cases_raw.sql` and `cases.sql` use `CREATE TABLE IF NOT EXISTS`. Does it matter?

Reply:
in this case, i think we do want to drop this table before proceeding -- if there's an existing `cases_raw_deduped` table when this SQL script runs, it will (probably) contain deduplicated rows from the previous run, which could be out of date. (and since we'll likely only be running the 01_cases task when the `cases_raw` table changes, it's very likely any existing `cases_raw_deduped` table would be stale.)